pkg/meta/meta.go

```diff
@@ -14,45 +14,17 @@
 package meta
 
 import (
+	"github.com/pingcap-incubator/tiup/pkg/localdata"
```

Comment:
Place all `std` library imports at the front of the 3rd will be more idiomatic.

Reply:
I wish goimports did this, or even better, gofmt.